src/toil/test/cwl/cwlTest.py

```diff
@@ -360,7 +363,10 @@ def test_run_conformance(self, batchSystem=None, caching=False):
             if batchSystem:
                 cmd.extend(["--batchSystem", batchSystem])
             if caching:
-                cmd.extend(['--', '--disableCaching="False"'])
+                cmd.extend(['--', '--disableCaching=False'])
```

Comment:
I want to keep it explicit because otherwise changing the default value of `--disableCaching` (again) will silently change what condition we're testing on one side.